samples/wrapper/Makefile

```diff
@@ -30,6 +30,11 @@ clean: distclean
 distclean:
 	rm -f $(PROGS) *.o libstdc++.a
 
+install: all
+	@echo "--- The example app $(PROGS) is not installed ---"
```

Comment:
Such kind of message will mislead the user, (s)he will decide that smth wrong happened during installation and it should be fixed somehow.